src/graph_notebook/notebooks/04-Machine-Learning/neptune_ml_utils.py

```diff
@@ -34,62 +36,69 @@ def load_configuration():
             iam = False
     return host, port, iam
 
+
 def get_host():
     host, port, iam = load_configuration()
     return host
 
+
 def get_iam():
     host, port, iam = load_configuration()
     return iam
 
+
 def get_training_job_name(prefix: str):
     return f'{prefix}-{int(time.time())}'
 
+
 def check_ml_enabled():
-    host, port, use_iam = load_configuration()  
-    response = signed_request("GET", url=f'https://{host}:{port}/ml/modeltraining', service='neptune-db')
+    host, port, use_iam = load_configuration()
+    response = signed_request("GET", url=f'https://{host}:{port}/ml/modeltraining', service='ml-db')
```

Comment:
Are we sure changing the service name here won't break anything?

Reply:
Ah good catch! I did a refactor of the package and it replaced some things it shouldn't have. this needs to be reverted